### PR TITLE
added donation page #7028

### PR DESCRIPTION
**Pull request by gautampanday**

### CODE REVIEW

**Comment by gautampanday** [comment]
donate

File: theme/static/css/style.css [hunk 1]
```diff
@@ -702,6 +702,60 @@ line-height: 70px;
   background: #2d2d2d;
   color: #cccccc;
 }
+
 .blog-section {
   padding: 25px 0 75px;
+}
+
+.doante-tab > li.active > a {
```
Comment (GauravButola) [inline theme/static/css/style.css:723]: Fix active tab

**Comment by gautampanday** [comment]
donate

File: theme/templates/donate.html [hunk 2]
```diff
@@ -0,0 +1,140 @@
...
+    <ol class="breadcrumb breadcrumb-background">
+      <li><a class="color-black" href="{{ SITE_ROOT }}index.html">{% trans %}Home{% endtrans %}</a></li>
+      <li class="color-black">{% trans %}Donate{% endtrans %}</li>
+    </ol>
+  </div>
+</section>
+<section class="section-space">
+  <div class="container">
+    <div class="col-md-12 grid-spaces">
+      <p class="text-center donate-heading">{% trans %}No matter they're big or small, all of them benefit from a powerful application that is free from any vendor lock ins.if you're already using tryton,please consider contributing to the project or support the Foundation so the project remains vibrant and continues growing.We're happy to accept direct donation via:{% endtrans %}</p>
```
Comment (GauravButola) [inline theme/templates/donate.html]: First letter capital after fullstop `.` and also space after comma `,` and fullstop `.`

File: theme/templates/donate.html [hunk 3]
```diff
@@ -0,0 +1,140 @@
...
+        <div class="tab-pane" id="gittip">
+          <div class="col-md-6 col-md-offset-3">
+            <iframe style="border: 0; margin: 0; padding: 0;" src="https://www.gittip.com/tryton/widget.html" width="48pt" height="22pt"></iframe>
+          </div>
+        </div>
+      </div>
+    </div>
+    <p class="text-center mailing-list">{% trans %}*If you want your donation to stay anonymous please say it so in donation comments{% endtrans %}</p>
+    <h2 class="text-center heading">{% trans %}Donors{% endtrans %}</h2>
+    <p class="text-center">{% trans %}The tryton Foundation wants to thank individuals and companies that finance it's activity.Donors are lsited in descending order using an algorithm that takes into account both the amount and the age of the donation over the last year.{% endtrans %}</p>
```
Comment (GauravButola) [inline theme/templates/donate.html]: space after `.`

File: theme/templates/donate.html [hunk 4]
```diff
@@ -0,0 +1,140 @@
...
+        <div class="tab-pane" id="settings">
+          <div class="col-md-6 col-md-offset-3">
+            <a href="http://flattr.com/thing/1093304/Tryton" target="_blank">
+              <img src="//api.flattr.com/button/flattr-badge-large.png" alt="Flattr this" title="Flattr this" border="0">
+            </a>
+          </div>
+        </div>
+        <div class="tab-pane" id="gittip">
+          <div class="col-md-6 col-md-offset-3">
+            <iframe style="border: 0; margin: 0; padding: 0;" src="https://www.gittip.com/tryton/widget.html" width="48pt" height="22pt"></iframe>
```
Comment (GauravButola) [inline theme/templates/donate.html]: gittip icon looks broken

**Comment by tarunbhardwaj** [comment]
Wire transfer is empty http://www.tryton.org/foundation/donations.html

File: theme/templates/donate.html [hunk 5]
```diff
@@ -0,0 +1,140 @@
...
+  </div>
+</section>
+<section class="section-space">
+  <div class="container">
+    <div class="col-md-12 grid-spaces">
+      <p class="text-center donate-heading">{% trans %}No matter they're big or small, all of them benefit from a powerful application that is free from any vendor lock ins. If you're already using tryton, please consider contributing to the project or support the foundation so the project remains vibrant and continues growing. We're happy to accept direct donation via:{% endtrans %}</p>
+      <br>
+      <div class="col-md-12 text-center">
+        <ul class="nav nav-pills doante-tab">
+          <li class="active"><a href="#home" data-toggle="pill"><span>Wire transfer</span> |</a></li>
```
Comment (tarunbhardwaj) [inline theme/templates/donate.html]: Fix href to wiretransfer same in all

File: theme/templates/donate.html [hunk 6]
```diff
@@ -0,0 +1,140 @@
...
+          <li><a href="#profile" data-toggle="pill"><span>PayPal</span> |</a></li>
+          <li><a href="#messages" data-toggle="pill"><span>BitCoin</span> |</a></li>
+          <li><a href="#settings" data-toggle="pill"><span>Flattr</span> |</a></li>
+          <li><a href="#gittip" data-toggle="pill"><span>Gittip</span></a></li>
+        </ul>
+      </div>
+      <!-- Tab panes -->
+      <div class="tab-content grid-spaces">
+        <div class="tab-pane active" id="home">{% trans %}Wire transfer {% endtrans %}</div>
```
Comment (tarunbhardwaj) [inline theme/templates/donate.html]: add details from http://www.tryton.org/foundation/donations.html

File: theme/templates/donate.html [hunk 7]
```diff
@@ -0,0 +1,151 @@
...
+        </div>
+      </div>
+    </div>
+    <p class="text-center mailing-list">{% trans %}*If you want your donation to stay anonymous please say it so in donation comments{% endtrans %}</p>
+    <h2 class="text-center heading">{% trans %}Donors{% endtrans %}</h2>
+    <p class="text-center">{% trans %}The tryton Foundation wants to thank individuals and companies that finance it's activity. Donors are lsited in descending order using an algorithm that takes into account both the amount and the age of the donation over the last year.{% endtrans %}</p>
+    <br>
+    1. gcoop - Cooperative de Software Libre<br>
+    2. James Norman<br>
+    3. Mars Guitars <br>
```
Comment (cedk) [inline theme/templates/donate.html]: This list is dynamically generated.
Using this resource: http://www.tryton.org//assets/donators.txt
You can find the JS that fill it: http://www.tryton.org/foundation/donations.txt

File: theme/templates/donate.html [hunk 8]
```diff
@@ -0,0 +1,169 @@
...
+          </div>
+        </div>
+      </div>
+    </div>
+    <p class="text-center mailing-list">{% trans %}*If you want your donation to stay anonymous please say it so in donation comments{% endtrans %}</p>
+    <h2 class="text-center heading">{% trans %}Donors{% endtrans %}</h2>
+    <p class="text-center">{% trans %}The tryton Foundation wants to thank individuals and companies that finance it's activity. Donors are lsited in descending order using an algorithm that takes into account both the amount and the age of the donation over the last year.{% endtrans %}</p>
+    <br>
+   <div id="donators-list">
+      <a href="/theme/assets/donators.txt">List</a>
```
Comment (tarunbhardwaj) [inline theme/templates/donate.html]: path should be relative

**Pull request merged**